src/state/CallViewModel.test.ts

```diff
@@ -792,6 +801,55 @@ it("should show at least one tile per MatrixRTCSession", () => {
   });
 });
 
+it("should disambiguate users with the same displayname", () => {
+  withTestScheduler(({ hot, expectObservable }) => {
+    const scenarioInputMarbles = "abcd";
+    const expectedLayoutMarbles = "abcd";
+
+    withCallViewModel(
+      of([]),
+      hot(scenarioInputMarbles, {
+        a: [],
+        b: [aliceRtcMember],
+        c: [aliceRtcMember, aliceDopplegangerRtcMember],
+        d: [aliceRtcMember, aliceDopplegangerRtcMember, bobRtcMember],
+        e: [aliceDopplegangerRtcMember, bobRtcMember],
```

Comment:
How hard would it be to add the other cases: `LTR/RTL` and display name formatted like an impersonation?

Reply:
Now covered!